lib/goom/src/tentacle3d.c

```diff
@@ -260,15 +259,27 @@ static void pretty_move (PluginInfo *goomInfo, float cycle, float *dist, float *
 		*rotangle = fx_data->rot = (tmp + 15.0f*fx_data->rot) / 16.0f;
 }
 
+static inline uint32_t color_multiply(uint32_t col1, uint32_t col2)
+{
+	uint8_t *color1 = (uint8_t *) &col1;
+	uint8_t *color2 = (uint8_t *) &col2;
+  uint32_t col;
+	uint8_t *color = (uint8_t *) &col;
+
+	*color = (*color1) * (*color2) / 256;
+	color++; color1++; color2++;
+	*color = (*color1) * (*color2) / 256;
+	color++; color1++; color2++;
+	*color = (*color1) * (*color2) / 256;
+	color++; color1++; color2++;
+	*color = (*color1) * (*color2) / 256;
+
+  return col;
```

Comment:
Can you remove the tabs there.

Reply:
I've made the tabs consistent in tentacle3d.c.